client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
@dechov in that case, it would mean that we're showing the user a total weight of 0.299 when they change back to the previous package? That doesn't sound exactly right to me, but if it's a widespread issue I can see the case for solving it in a different PR.